lib/docker2aci.go

```diff
@@ -127,146 +209,203 @@ func GetRepoData(indexURL string, remote string) (*RepoData, error) {
 	}, nil
 }
 
-func GetRemoteImageJSON(imgID, registry string, repoData *RepoData) ([]byte, int, error) {
+func getImageIDFromTag(registry string, appName string, tag string, repoData *RepoData) (string, error) {
 	client := &http.Client{}
-	req, err := http.NewRequest("GET", registry+"images/"+imgID+"/json", nil)
+	req, err := http.NewRequest("GET", "https://"+path.Join(registry, "repositories", appName, "tags", tag), nil)
 	if err != nil {
-		return nil, -1, err
+		return "", fmt.Errorf("Failed to get Image ID: %s, URL: %s", err, req.URL)
```

Comment:
errors should always start lowercase (in anticipation of being printed by callers with a prefix)